Modules/IndividualAssessment/classes/class.ilIndividualAssessmentMembersGUI.php

```diff
@@ -28,13 +28,14 @@ public function __construct($a_parent_gui, $a_ref_id) {
 		$this->tpl =  $DIC['tpl'];
 		$this->lng = $DIC['lng'];
 		$this->toolbar = $DIC['ilToolbar'];
-		$this->access_handler = $this->object->accessHandler();
+		$this->iass_access = $this->object->accessHandler();
```

Comment:
iass has his own access handler class. Rename makes clear this is not the global ilias access handler.